### DISCUSSION TITLE
[admin-boundaries] US county data download

### DISCUSSION THREAD

**Discussion by iamlaksh1** [OvertureMaps/data#212] [Discussion]
### Category Feedback

Hello

I am aware I can use CLI to download data using bounding box. Is there any way to download all themes for selected US county?

**Comment by Krizz** [comment]
I've created a tool for extracting Overture layer data by location:
https://github.com/Krizz/fetch_overture

**Reply by iamlaksh1** [reply]
Amazing. Let me try it today !!
